sql/src/parser.rs

```diff
@@ -540,6 +543,130 @@ fn build_timestamp_key_constraint(col_defs: &[ColumnDef], constraints: &mut Vec<
     }
 }
 
+/// Add quotes in table name (for example: convert table to `table`).
+///
+/// It is used to process table name in `SELECT`, for preventing `datafusion`
+/// converting the table name to lowercase, because `CeresDB` only support
+/// case-sensitive in sql.
+// TODO: maybe other items(such as: alias, column name) need to be normalized,
+// too.
+pub fn maybe_normalize_table_name(statement: SqlStatement) -> SqlStatement {
+    let original_statement = statement.clone();
+    if let SqlStatement::Query(query) = statement {
```

Comment:
Maybe we can modify the statement directly to avoid verbose struct unwrapping.

Reply:
> Maybe we can modify the statement directly to avoid verbose struct unwrapping.

Done.